packages/manager/modules/advices/src/advices/translations/Messages_fr_FR.json

```
@@ -1,3 +1,4 @@
 {
-  "advices_heading": "Nos conseils"
+  "advices_heading": "Nos conseils",
+  "advices_retention_heading": "Nos guides"
```

I know you don't touch that, but regarding others languages w have to add:
 ```suggestion
  "advices_retention_heading": "Nos guides :"
```

---

The content is reviewed and approved. There is no semicolon at the end. I am not sure why translation team added it in other languages when its not there in source language.

---

Ok, so we should probably create another task to remove the extra semicolon for others languages.